rules.txt

```diff
@@ -0,0 +1,148 @@
+Panorama geral do projeto, vamos planejar a partir disso:
+
+## Unidades Espaciais ##
+- Fighters / Bombers
+  Podem dar jump a planetas proximos que tenham CS aliada
+  Diferenciados pelo uso de misseis ou torpedos
+
+- CapitalShips
+  Carregam Fighters/Bombers/Weapons/Troopers/Vehicles
+  Podem ter upgrades e/ou XP
+  Podem dar hyperjumps distantes
+
+- Transports
+  Carregam Fighters/Bombers/Troopers/Vehicles (Em menor qtde do q uma CS)
+  Podem dar hyperjumps planetas próximos
+
+- Shipyards
+  Constroem as naves em turnos
+  Tem "pontos de produção" para construir as unidades
+
+## Unidades Terrestres ##
+- HeadQuarter (NOVO)
+  Barrack quartel general do Squad
+  Só pode haver uma
+  Produz tropas e veículos
+  Abriga Heroes
+
+- Barracks (NOVO)
+  Produz tropas e veículos
+
+- Troopers
+  Soldados para ataques terrestres
+
+- Vehicles
+  Veiculos Terrestres. AT-AT, AT-ST etc
+
+## Unidades Especiais ## 
+- Heroes
+  Cumprem missões diversas. Espionagem, Sabotagem, Diplomacia etc
+
+## Produção das Shipyards e Barracks ##
+- Sua construção depende de créditos/metais e % de dominação do planeta
+-  Ideal poder construir baseado na sua capacidade e setando por exemplo: 20 X-Wings, 50% Calamari Cruiser, 30% Transports e ela ir produzindo sozinho. 
+-  Opção também para comprar unidades instantaneamente com créditos/metais
+
+## Movimentos ##
+   Tem q ser user friendly. Arrastar e soltar ele pede a qtde a ser movida, mostra o tempo de viagem em turnos baseado no algoritmo Dikstra.
```

Comment:
Tenho minhas duvidas se arrastar e soltar seria a parada. Tenta fazer um experimento com essa interface so no frontend fingindo que a nave tem varias coisas. A ideia eh: se vc quer tentar fazer isso, simula so no HTML isso acontecendo pra gente ver como fica e ver se vale a pena.

Reply:
Nao tenho a menor ideia de como fazer isso rsssss. Mas pense q a interface é sem sombra de duvidas o maior gargalo atualmente. Pode ser daóra o jogo mas se for feio ou complicado, ninguem quer jogar